src/main/java/WayofTime/bloodmagic/item/ItemBindableBase.java

```diff
@@ -24,11 +24,11 @@ public ItemBindableBase() {
     @Override
     @SideOnly(Side.CLIENT)
     public void addInformation(ItemStack stack, World world, List<String> tooltip, ITooltipFlag flag) {
-        if (!stack.hasTagCompound())
-            return;
-
-        Binding binding = getBinding(stack);
-        if (binding != null)
-            tooltip.add(TextHelper.localizeEffect("tooltip.bloodmagic.currentOwner", binding.getOwnerName()));
+        if (stack.hasTagCompound()) {
```

Comment:
There is no reason for these formatting changes

Reply:
I made it to prevent code duplication when I added a tooltip, I'll revert it now since said tooltip is not relevant anymore.